libs/core/src/components/pds-copytext/pds-copytext.tsx

```diff
@@ -41,24 +41,12 @@ export class PdsCopytext {
 
   private copyToClipboard = async (value: string) => {
     try {
-      if (navigator.clipboard) {
+      if (typeof navigator.clipboard !== 'undefined') {
         await navigator.clipboard.writeText(value);
         this.pdsCopyTextClick.emit('Copied to clipboard');
-      } else {
-        // falback for safari
-        const el = document.createElement('textarea');
-        el.value = value;
-        el.setAttribute('readonly', '');
-        el.style.position = 'absolute';
-        el.style.left = '-9999px';
-        document.body.appendChild(el);
-        el.focus();
-        el.setSelectionRange(0, el.value.length);
-        document.execCommand('copy');
-        document.body.removeChild(el);
-        this.pdsCopyTextClick.emit('Copied to clipboard');
```

Comment:
Did you check to make sure the component still works in Safari? I didn't want to make any assumptions, hence the question.

Reply:
Yes, all the browsers tested for this are mentioned in the config section.